src/logic/map_objects/tribes/tribe_descr.h

```diff
@@ -190,6 +194,13 @@ class TribeDescr {
 
 	uint32_t frontier_animation_id_;
 	uint32_t flag_animation_id_;
+	uint32_t bridge_e_animation_normal_id_;
```

Comment:
I'm wondering whether these would be easier to read if we stuck them in a struct, one for the normal ones and one for the busy ones. If you don't think it would be better, just resolve this comment.

Reply:
Ok